shared/constants/permissions.ts

```diff
@@ -32,6 +42,33 @@ export const EndowmentPermissions = Object.freeze({
 } as const);
 
 // Methods / permissions in external packages that we are temporarily excluding.
-export const ExcludedSnapPermissions = new Set([]);
-export const ExcludedSnapEndowments = new Set(['endowment:keyring']);
+export const ExcludedFlaskSnapPermissions = {
+  eth_accounts:
+    'eth_accounts is disabled. For more information please see https://github.com/MetaMask/snaps-monorepo/issues/990.',
+};
+export const ExcludedStableSnapPermissions = {
+  eth_accounts:
+    'eth_accounts is disabled. For more information please see https://github.com/MetaMask/snaps-monorepo/issues/990.',
+};
+export const ExcludedStableSnapEndowments = {
+  'endowment:keyring': 'This endowment is not available',
+  'endowment:long-running':
+    'endowment:long-running is deprecated. For more information please see https://github.com/MetaMask/snaps-monorepo/issues/945. ',
```

Comment:
Please make sure these issue links actually explain why the snap endowment is disabled or deprecated

Reply:
It's using the same issue as the warning in snap-monorepo. It seems to be the most relevant issue regarding the deprecation of `endowment:long-running`.

Reply:
Yeah, I just meant make sure to leave a comment or update the issue to make sure it correctly explains why we are disabling it. I does not have a lot of information at this point.

Reply:
Will leave a message on the linked issue :)

Reply:
hey, unresolving this as I didn't see an update to the issue or a comment.  Same for #990.

Reply:
Yes I just wanted to wait for this to be merged so we don't create confusion.